fastchat/model/model_adapter.py

```diff
@@ -616,11 +615,13 @@ def get_default_conv_template(self, model_path: str) -> Conversation:
         return get_conv_template("vicuna_v1.1")
 
 
-class CodeT5pAdapter(BaseModelAdapter):
-    """The model adapter for Salesforce/codet5p-6b"""
+class GoogleFlanAdapter(BaseModelAdapter):
+    """The model adapter for google/Flan based models, such as Salesforce/codet5p-6b, lmsys/fastchat-t5-3b-v1.0, flan-t5-*, flan-ul2"""
 
     def match(self, model_path: str):
-        return "codet5p" in model_path.lower()
+        return any(
+            model_path in model_str for model_str in ["flan-", "fastchat-t5", "codet5p"]
```

Comment:
I think the correct one should be. Did you test your code?
```suggestion
            model_str in model_path.lower() for model_str in ["flan-", "fastchat-t5", "codet5p"]
```

Reply:
I think you are right.